deps/v8/src/debug-debugger.js

```diff
@@ -430,7 +430,7 @@ ScriptBreakPoint.prototype.set = function (script) {
   if (IS_NULL(position)) return;
 
   // Create a break point object and set the break point.
-  var break_point = MakeBreakPoint(position, this);
+  break_point = MakeBreakPoint(position, this);
```

Comment:
We may want to float https://github.com/v8/v8-git-mirror/commit/2ad2237507c5b5f9047b8d94d2f4997327eae852 again to avoid issues with `--debug-brk` when used with `--use-strict`.